go.mod

```diff
@@ -59,6 +59,8 @@ require (
 	gopkg.in/macaroon-bakery.v2 v2.0.1
 	gopkg.in/macaroon.v2 v2.0.0
 )
+// This replace is to leverage updated batch-json rpc
+replace github.com/btcsuite/btcwallet => ../btcwallet
```

Comment:
A better way for this replace rule is to reference your pushed branch, so something like:

```
replace github.com/btcusite/btcwallet =? github.com/Vib-UX/btcwallet hash
```

Where the hash is your HEAD's hash. Then you do a `go mod tidy` and it should fetch the reference correctly. This way if someone checks out your branch they can build it without fixing these deps first.